static/js/main.js

```diff
@@ -241,17 +243,23 @@ function addScreenshot(screenshot) {
                             value='${time_input_value}' />
                 </div>
                 <div>
-                    <button onclick='flipImage("${screenshot.id}")'>Flip Image</button>
+                    <span>
+                        <button onclick='flipScreenshot("${screenshot.id}", axis="x")'>Vertical Flip</button>
```

Comment:
Does this line actually work?
Sending an argument by name is not universally supported:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters

I would just not bother making it optional.

Reply:
Ok, changed it, if was working on Chrome.

Reply:
Remember that all comments should be resolved or commented on if it needs further discussion.